src/Sarif.Multitool/Rules/SARIF1021.MissingToolVersion.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using Microsoft.Json.Pointer;
+
+namespace Microsoft.CodeAnalysis.Sarif.Multitool.Rules
+{
+    public class MissingToolVersion : SarifValidationSkimmerBase
+    {
+        private readonly MultiformatMessageString _fullDescription = new MultiformatMessageString
+        {
+            Text = RuleResources.SARIF1021_MissingToolVersion
+        };
+
+        public override MultiformatMessageString FullDescription => _fullDescription;
```

Comment:
>FullDescription [](start = 49, length = 15)

You should recapitulate the full description as a doc comment on this method, because it's helpful for rules readability. See the following BinSkim rule for thoughts on a standard in terms of commenting, organization, etc. 

[https://github.com/microsoft/binskim/blob/master/src/BinSkim.Rules/PERules/BA2010.DoNotMarkImportsSectionAsExecutable.cs](https://github.com/microsoft/binskim/blob/master/src/BinSkim.Rules/PERules/BA2010.DoNotMarkImportsSectionAsExecutable.cs)